### DISCUSSION TITLE
Server Error - Unable to retrieve cluster info

### DISCUSSION THREAD

**Discussion by bl7711** [rackslab/Slurm-web#385] [Q&A]
Slurm-Web unable to retrieve cluster info with Server Error Message.

Gateway seemed to have issue processing response from agent?  Any help is very much appreciated.

[agent-log.txt](https://github.com/user-attachments/files/17709705/agent-log.txt)
[gatweway-log.txt](https://github.com/user-attachments/files/17709706/gatweway-log.txt)

**Comment by rezib** [comment]
Hello @bl7711, this is clearly a bug, I just opened #386 to fix it. Thank you for your report! :+1:
